### DISCUSSION TITLE
Speeding up ESLint—Even on CI

### DISCUSSION THREAD

**Discussion by giscus** [charpeni/charpeni.com#91] [Comments]
# Speeding up ESLint—Even on CI

Gotta Cache 'Em All: Leverage ESLint's cache to speed it up.

https://www.charpeni.com/blog/speeding-up-eslint-even-on-ci

**Comment by kurtextrem** [comment]
Thank you!

**Comment by xiongemi** [comment]
would it generates unwanted merge conflicts if too many developers work on `.eslintcache`

**Reply by charpeni** [reply]
Yes.

`.eslintcache` should be unique for each machine, that's why we should [add it to `.gitignore`](https://www.charpeni.com/blog/speeding-up-eslint-even-on-ci#:~:text=We%20also%20need,%3E%3E%20.gitignore).
